### ISSUE TITLE
Need util for applying temporary immunity to a single weapon damage type

### ISSUE THREAD

**Issue by 59blargedy** [project-topaz/topaz#589]
**_I have:_**

- [x] searched existing issues (http://project-topaz.com/issues/) to see if the issue has already been opened
- [x] checked the commit log to see if the issue has been resolved since my server was last updated
Need some way of cleanly applying temporary resistances and removing them. can currently addmobmod PIERCERES etc, but then need calls to remove.

reference: https://github.com/project-topaz/topaz/pull/561#discussion_r420708141
this could also affect mobs like Jailer of Temperance, Gladiatoral Weapons in Die By the Sword BCNM, wamouracampa, to name a few.

**Comment by TeoTwawki** [comment]
yeah the current way we could do this in script result sin a lot of repeated messy code..Could def use utils function to clean it up and bake in handling of related things.

Edit: _if anyone doubts the pain of doing it with current methods, I have a custom mob that rotates weaknesses and winds up many many lines long you can witness and then regret looking at ^^; table/array inputs for what resistances to manipulate would be super._
